Source/inv.cpp

```diff
@@ -887,7 +887,7 @@ void CheckInvCut(int pnum, Point cursorPosition, bool automaticMove, bool dropIt
 		}
 	}
 
-	if (dropItem) {
+	if (dropItem && holdItem._itype != ItemType::None) {

```

Comment:
```suggestion
	if (dropItem && !holdItem.isEmpty()) {
```

The Item struct provides a convenience method for performing this check

Reply:
Thank you @ephphatha I didn't notice.